.travis.yml

```diff
@@ -47,6 +47,17 @@ jobs:
       language: python
       script:
         - make cfn/version VERSION=$RELEASE_VERSION
+    - stage: test
+      env:
+        - JOB="Module Docs Lint"
```

Comment:
We can switch all these jobs to use the `name` option instead of this JOB env... e.g.

```
- stage: test
  name: Module Docs Lint
```